src/components/styles/Homepage.css

```diff
@@ -0,0 +1,60 @@
+body {
+  box-sizing: border-box;
+  background-color: #e2588a;
+  color: white;
+}
+
+.mainHeader {
+  display: flex;
+  margin-bottom: 5%;
+  justify-content: center;
+  gap: 7%;
+}
+
+.mainHeader img {
+  width: 100px;
+  height: auto;
+}
+
+.arrowRegion {
+  width: 25px;
+  height: auto;
+}
+
+.countryList {
+  list-style: none;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  font-weight: bold;
+}
+
+.input {
+  border-radius: 8px;
+  padding: 10px;
+  background: #fff;
+  width: 100%;
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.subHeader {
+  margin: 0;
+  padding: 0;
+  font-size: small;
+}
+
+.listItem {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding: 10px 20px;
+}
+
+.listItem:nth-child(3n-1) {
+  background-color: #d04e79;
+}
```

Comment:
- [ ] [OPTIONAL] This part of the styling can be improved a bit, you can do a similar thing to the second nth-child to make it look exactly like the original.

Reply:
@elisha2kyakpo1 Thank you so much for your suggestion. But I didn't understand what you meant exactly.